ksql-rest-app/src/main/java/io/confluent/ksql/api/server/ApiServerConfig.java

```diff
@@ -168,6 +156,9 @@ public ApiServerConfig(final Map<?, ?> map) {
     super(CONFIG_DEF, map);
   }
 
+  // Note that this expects config value that is not standard Confluent as defined in BaseConfig.
+  // For "request" client auth, confluent value is "REQUESTED" where value expected here is
```

Comment:
Out of curiosity, how come we chose "request" instead of "requested"? Seems to break parallelism with "required" which was preserved.

Reply:
This is because of the Vert.x ClientAuth enum:

https://github.com/eclipse-vertx/vert.x/blob/master/src/main/java/io/vertx/core/http/ClientAuth.java

which uses "REQUIRE" and we map directly from strings to the enum.

We should probably make this consistent with the standard Confluent config (i.e. do a manual mapping from string to Vert.x enum)